__tests__/__src__/environment/TestEnvironment.ts

```diff
@@ -52,7 +52,10 @@ export class TestEnvironment {
         // Get a unique test data area
         const testDirectory: string = TestEnvironment.createUniqueTestDataDir(params.testName);
 
-        const systemProps = TestEnvironment.loadSystemTestProperties(undefined, testDirectory);
+        let systemProps;
+        if (!params.skipProperties) {
+            TestEnvironment.loadSystemTestProperties(undefined, testDirectory);
```

Comment:
Looks good

Reply:
I messed up here since I didn't even assign the systemProps variable.
Whoops. 
Will fix it now

Reply:
Haha whoops